src/Surfnet/Stepup/Configuration/Entity/RaLocation.php

```diff
@@ -0,0 +1,140 @@
+<?php
+
+/**
+ * Copyright 2016 SURFnet B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Surfnet\Stepup\Configuration\Entity;
+
+use Surfnet\Stepup\Configuration\Value\ContactInformation;
+use Surfnet\Stepup\Configuration\Value\Location;
+use Surfnet\Stepup\Configuration\Value\RaLocationId;
+use Surfnet\Stepup\Configuration\Value\RaLocationName;
+
+class RaLocation
+{
+    /**
+     * @var RaLocationId
+     */
+    private $raLocationId;
+
+    /**
+     * @var RaLocationName
+     */
+    private $raLocationName;
```

Comment:
@rjkip / @arothuis why is this ```raLocationId``` instead of ```id``` of the ```RaLocation``` object? Does this mean the RA will have to specify  the ```orderBy``` as ```ra_location_id``` instead of ```id```?